include/revng/Model/NameBuilder.h

```diff
@@ -194,6 +195,12 @@ class NameBuilder {
     return Function.Name();
   }
 
+  /// This is just here for compatibility: it's nice to be able to pass anything
+  /// into the name builder and get the name back.
```

Comment:
This more a comment than Doxygen

Reply:
Done, and while at it, I also added a forgotten assert to it:
```cpp
  // This is just here for compatibility: it's nice to be able to pass anything
  // into the name builder and get the name back.
  std::string name(const model::PrimitiveType &Primitive) const {
    std::string Result = Primitive.getCName();
    assertNameIsReserved(Result);
    return Result;
  }
```